spec/lib/services/sku-pack-service-spec.js

```diff
@@ -107,8 +117,22 @@ describe("SKU Pack Service", function() {
                 destroy: sinon.stub()
             };
 
+            waterline.templates = {
```

Comment:
Not sure what you want restored?  There is no 'waterline.<model>' when this is run, which is why I create them.  I cannot restore that which doesn't exist.